docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -90,11 +90,11 @@ than one service instance consideration must be given to the naming of objects.
 
 Separate the public and private configuration information in your APB.
 
-Use *configMaps* for public information that a mobile client will need to use the service. To allow the configMap
-to be read by the mobile-cli it must be labeled with *mobile: enabled* and the *serviceName*.
-
+Use *configMaps* for public information that a mobile client will need to use the service. 
```

Comment:
Should  this be \`configMaps\` rather than \*configMaps\*?

Reply:
Thanks, updated *secrets* as well